### DISCUSSION TITLE
Submit to Frigate+ not working

### DISCUSSION THREAD

**Discussion by matknowles** [blakeblackshear/frigate#14422] [Ask A Question] [answered]
I've been submitting images (correct and false positives) through the homeassistant/Frigate/Submit to Frigate+ page in Edge, but I've noticed that the images count on my frigate+ dashboard isn't going up when I refresh. Can someone tell me if this is to do with a config or IP/VPN issue or something else? I have a couple hundred images uploaded that I've done previously over the last couple of months, but for example today I spent a while going through images to submit and also noticed that the same images kept popping up to submit, despite me having submitted already for that image. I have encountered this issue before a few weeks ago, but didn't bother to ask the question here. I am using a VPN.

**Comment by NickM-27** [comment]
there should be an error popup if the submission failed

**Comment by matknowles** [comment]
Thanks for that. I've tried on another machine, in Firefox and also without a VPN and the issue persists. If I reload the frigate+ submission page, the images I've already submitted reappear and the counts in my frigate+ dashboard don't change. I'm not seeing anything popup when I submit and ublock isn't registering any blocked domains (and disabling it doesn't make any difference). Is there a token perhaps or something that has expired? I think all I've done is added my model ID to the frigate config.yaml. Can't see what I might be doing wrong, but I'm having some issues on a couple of cameras with false positives so I want to be able to refine my model. I wonder if re-generating my API key and changing that in home assistant will fix it?

**Reply by NickM-27** [reply]
So you get a 200 response from the request to send the image?

**Reply by matknowles** [reply]
I don't know, I'm just doing the submission through a browser. There's something in the logs though: 2024-10-18 13:11:03.038740895  [ WARN:11302@4433.483] global loadsave.cpp:248 findDecoder imread_('/media/frigate/clips/Shed-1729256964.130061-1f8q9l-clean.png'): can't open/read file: check file path/integrity
2024-10-18 13:11:03.039581778  [2024-10-18 14:11:03] frigate.api.event              ERROR   : Unable to load clean png for event: 1729256964.130061-1f8q9l
2024-10-18 13:11:03.040184074  172.30.32.2 - - [18/Oct/2024:14:11:03 +0100] "PUT /api/events/1729256964.130061-1f8q9l/false_positive HTTP/1.1" 400 65 "http://homeassistant.local:8123/api/hassio_ingress/xxxxxxx/plus" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36 Edg/129.0.0.0" "192.168.1.30, xxx.xx.xx.xx"

**Reply by matknowles** [reply]
I am using frigate through NAS connected storage as the SSD I'm running HAOS on isn't big enough for my retention policy. Wonder if that's something to do with it?

**Reply by NickM-27** [reply]
Are you sure you have clean copy enabled in the config?

**Reply by matknowles** [reply]
My config shows: snapshots:
  enabled: true
  clean_copy: false
  timestamp: false
  bounding_box: true

so do i need to enable that? Don't remember this being an issue in the past.

**Reply by NickM-27** [reply, marked as answer]
Clean copy has always been required

**Reply by matknowles** [reply]
OK that's great. Thanks for the clarification, It didn't fix it until I rebooted the system. I must've changed that option at some point as I didn't want clean snapshots (wanted to see what Frigate was detecting for debug/tuning purposes.)

Fixed anyway, thanks again.

**Comment by matknowles** [comment]
I've changed the API key, reloaded and the issue persists unfortunately.
